src/components/show-products/index.js

```diff
@@ -43,27 +67,46 @@ export const ShowProducts = () => {
 
   return (
     <ContentContainer>
+      <form className="filter-form">
+        <label htmlFor="filter">
+          Filter items
+          <input
+            className="input-item"
+            type="search"
```

Comment:
It's so great that you've used an input `search`: 💯 .

However, I would like to say two things (maybe for the future) 😄:
1. Check browser compatibility (Google Chrome, Mozilla Firefox, MS Edge, and so on) because this kind of input has [a little issue in Firefox.](https://phabricator.services.mozilla.com/D56302). An input search usually has a cross icon on the right side, but unfortunately, Firefox has [this feature disabled] 😢 (https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Experimental_features#layout_for_input_typesearch).

![image](https://user-images.githubusercontent.com/8689897/140445438-28a528f7-41ab-4adb-a6a0-3f8a3cb38908.png)

Maybe in this specific situation, there isn't a problem or requested changes because ACs were so detailed or mentioned about browsers supported. 👍 

2. Possible solutions

- Create a button `X`
- Working with CSS like [this](https://codepen.io/shidhincr/pen/ICLBD).

Reply:
I solved this problem with the following commit 7c0f11d